srcpkgs/howl/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'howl'
+pkgname=howl
+version=0.6
+revision=1
+archs="i686 x86_64 x86_64-musl"
+build_wrksrc=src
+build_style=gnu-makefile
+make_use_env=yes
+hostmakedepends="wget gcc pkg-config gtk+3"
+makedepends="gtk+3-devel"
+depends="gtk+3"
+short_desc="Lightweight and fully customizable general purpose editor"
+maintainer="reback00 <reback00@protonmail.com>"
+license="MIT"
+homepage="http://howl.io/"
+distfiles="https://github.com/howl-editor/howl/releases/download/${version}/howl-${version}.tgz"
+checksum=834b06e423d360c97197e7abec99b623fdc5ed3a0c39b88d6467e499074585e1
+python_version="3"
+
+post_install() {
+	vlicense ../LICENSE.md
```

Comment:
Isn't the _LICENSE.md_ file located in the root dir of the source tarball?

Reply:
Because it gets run in `src/`, it needs to look one directory up for it

Reply:
I think it has to like this due to having `build_wrksrc` being [required to be](https://github.com/void-linux/void-packages/pull/21717#discussion_r443182693) set to `src`.
Just for reference, having `vlicense LICENSE.md` directly, raises this error:

```
...
=> howl-0.6_1: running post_install ...
=> ERROR: howl-0.6_1: vinstall: cannot find 'LICENSE.md'...
=> ERROR: howl-0.6_1: post_install: 'return 1' exited with 1
...
```